src/contracts/core/AVSDirectory.sol

```diff
@@ -625,13 +628,15 @@ contract AVSDirectory is
         IStrategy strategy,
         OperatorSet calldata operatorSet
     ) internal view returns (uint64 freeMagnitudeToAdd) {
```

Comment:
doesn't even touch this return var?

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/680/commits/d33bceec29869d0ee995a73cefc5c98e08d72c27